python/paddle/v2/fluid/distribute_transpiler_simple.py

```diff
@@ -243,7 +243,7 @@ def get_pserver_program(self, endpoint, optimize_ops):
                     self.param_grad_map[endpoint]["grads"]},  # grads to recv
             outputs={},
             attrs={
-                "OptimizeProgram": optimize_sub_program.desc,
+                "OptimizeBlock": optimize_sub_program.global_block(),
```

Comment:
We can merge `optimize_sub_program` to the `pserver_program` as the second block (block_id 1), then we don't need to create variables in both two of the programs.